modules/vpc/variables.tf

```diff
@@ -1,8 +1,3 @@
-variable "azs" {
```

Comment:
Should remove it from variables.tf as well.

Reply:
Removed.